ts/helpers/attributesHandler.ts

```diff
@@ -0,0 +1,29 @@
+
+export const getAttributeByPath = (slideAttributes, path) => {
+
+    if(!Array.isArray(path)){
```

Comment:
use strict comparison when return type and value is exactly known

```suggestion
    if (Array.isArray(path) === false) {
```

Reply:
done